nautobot/docs/rest-api/ui-related-endpoints.md

```diff
@@ -0,0 +1,8 @@
+# UI Related API Endpoints
+
++++ 2.0.0
+
+Nautobot now retrieves its UI component information from API endpoints.
+
+!!!important
+    Note that these API endpoints are hidden from OpenAPI schema and do not support REST API versioning.
```

Comment:
Let's also make it very explicit that these endpoints are NOT supported and are for internal use only and therefore are not subject to SemVer and can change at any time to support the UI.

That is of course, until such point that we decide we are ready to support any UI-specific endpoints.

While on this topic: We should be very deliberate in only having UI-specific endpoints as an exception and not the norm. Non-UI REST API endpoints should continue to be utilized wherever possible.